src/Model.php

```diff
@@ -710,6 +710,7 @@ public function addCondition($field, $operator = null, $value = null)
 
         if ($f) {
             $f->system = true;
+            $f->ui['editable'] = false;
```

Comment:
I think unless you set editable to true/false explicitly, it falls back to 'system' here:

https://github.com/atk4/data/blob/develop/src/Field.php#L266

Does that not work?